### ISSUE TITLE
Two lines of same group requires I201

### ISSUE THREAD

**Issue by rafis** [PyCQA/flake8-import-order#154]
The following two lines are detected as same group that is 'Third Party'. Why it requires to put a newline between them?
```python
from django_telegrambot.apps import DjangoTelegramBot
from telegram.ext import CommandHandler, Filters, MessageHandler
```

Error message:

.\bot\telegrambot.py:2:1: I201 Missing newline between import groups. 'from telegram.ext import CommandHandler, MessageHandler, Filters' is identified as Third Party and 'from django_telegrambot.apps import DjangoTelegramBot' is identified as Third Party.

**Comment by pgjones** [comment]
This is the definition of the Cryptography [style](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_cryptography.py) you can chose another if you disagree.
